src/main/java/org/abstractj/kalium/crypto/Stream.java

```diff
@@ -0,0 +1,42 @@
+package org.abstractj.kalium.crypto;
+
+import static org.abstractj.kalium.NaCl.sodium;
+import static org.abstractj.kalium.crypto.Util.checkLength;
+import static org.abstractj.kalium.crypto.Util.isValid;
+import static org.abstractj.kalium.crypto.Util.prependZeros;
+import static org.abstractj.kalium.crypto.Util.removeZeros;
+
+import org.abstractj.kalium.NaCl;
+
+public class Stream {
+
+	private byte[] key;
+
+	public Stream(byte[] key) {
+		this.key = key;
+		checkLength(key, NaCl.Sodium.AES_128_CTR_KEYBYTES);

```

Comment:
Make use of import static, please. Into this way we just leave the constant into the code.